conda/recipes/libcudf/meta.yaml

```diff
@@ -40,7 +41,7 @@ requirements:
     - arrow-cpp 0.17.1.*
     - dlpack
   run_constrained:
-    - arrow-cpp-proc * 0.17.1.*
+    - arrow-cpp-proc * {{ build_ext }}
```

Comment:
I think this should actually be `- arrow-cpp-proc * cuda` but I could be mistaken. https://anaconda.org/conda-forge/arrow-cpp-proc/files

It's always version 1.0.0 and the build string is either `cpu` or `cuda`.